### ISSUE TITLE
Comments and relationship block issue

### ISSUE THREAD

**Issue by ruddell** [jhipster/jhipster-core#59]
The following JDL is considered invalid due to the comment in the relationship block.  It throws the error `Expected ",", "}" or a space but "/" found.`.  Comments in this format work elsewhere and are found in the documentation.
```
entity Book
entity Author
relationship OneToMany {
  // this comment is OK
  Author{book} to Book{writer(name)} // this comment breaks the generation!
  // this comment is also OK
}
```

**Comment by MathieuAA** [comment]
Nice catch! I'll fix it for v1.1.10. Thanks Jon!

**Comment by MathieuAA** [comment]
Oh and by the way, it won't work as it will expect to find a `name` attribute in the Author entity =) but the bug stands.
